phys2bids/phys2bids.py

```diff
@@ -296,6 +297,12 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
                       phys_in.num_timepoints_found, uniq_freq,
                       phys_out[uniq_freq].start_time, outfile)
 
+    # Generate participants.tsv file if it doesn't exist already.
+    # Update the file if the subject is not in the file.
+    # Do not update if the subject is already in the file.
+    if heur_file and sub:
```

Comment:
I am gonna insist, there is a if statement that is identical to this one in line 266, let's not duplicate code. I know you wanted to put the participants.tsv code at the end of the function but why not there? it doesn't influence the rest of the code

Reply:
As I previously said, line 266 is part of a for loop. I don't think it makes sense to run `participants_file(indir, outdir, yml, sub)` for every frequency in the file.

Reply:
Resolving this after seeing your comment above.